examples/pyfunc/conda.yaml

```diff
@@ -0,0 +1,9 @@
+name: pyfunc_model
+channels:
+  - conda-forge
+dependencies:
+  - python=3.7
+  - pip=20.0.2
+  - pip:
+    - mlflow>=1.12
+    - scikit-learn==1.1.1
```

Comment:
@sniafas Scikit-learn 1.1 requires Python 3.8 or newer. I think we can just remove the version specifier.